blue-green-upgrade/cluster_ops.sh

```diff
@@ -149,16 +149,16 @@ install_app() {
   kubectl -n default create -f "${REPO_HOME}/manifests/es-svc.yaml"
   kubectl -n default create -f "${REPO_HOME}/manifests/es-master-pdb.yaml"
   kubectl -n default create -f "${REPO_HOME}/manifests/es-master.yaml"
-  kubectl -n default rollout status -f "${REPO_HOME}/manifests/es-master.yaml"
+  kubectl -n default rollout status --timeout=2m -f "${REPO_HOME}/manifests/es-master.yaml"
```

Comment:
We should eventually loop here if this fails.  But this work for now?

Reply:
Yes, it works for now. I added a buffer of extra '1m' as you said sometimes cloud can act crazy.